Book/tests/running_the_test_suite.rst

```diff
@@ -90,7 +90,7 @@ You can specify multiple target folders or files to run by separating each path
 Executing tests in parallel
 ---------------------------
 
-You can't.
+Since PHP 7.4 you are able to run tests in parallel by passing a ``-j<workers>`` flag, where workers is the number of parallel processes you want. e.g. ``-j4`` would run 4 tests at once.
```

Comment:
I think this should clarify that you need to pass `-j` to run-tests.php and not make, either by directly calling `run-tests.php -jN` or by using `make test TESTS="-jN"`.

Reply:
I never know who should click the resolve conversation button.